### PR TITLE
Request code review for Player class tests

### PR DESCRIPTION
**Pull request by KaushikShivam**

### CODE REVIEW

**Comment by KaushikShivam** [comment]
We've tried to fix all the stickler errors. Only 4 are not passing and all of them relate to our blocks being too long. We can't fix it as we are being comprehensive about our tests and the specs/examples should only stay inside the describe/example group. Looking forward to your feedback.

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    end
+  end
+
+  describe '#check_vertical' do
+    it 'should return true if the slots match vertically from first column' do
+      allow(game).to receive(:matrix)
+        .and_return([%w[x _ o], %w[x _ o], %w[x _ _]])
+      expect(game.check_vertical).to eql(true)
+    end
+    it 'should return true if the slots match vertically from second column' do
```
Comment (bellom) [inline spec/game_spec.rb:51]: Always Leave one empty line between feature, context or describe blocks. Do not leave empty line after the last such block in a group. [checkout this article to fix the issue](https://github.com/rubocop-hq/rspec-style-guide#empty-lines-between-describes)
kindly apply to the rest of the `it` block without space.

File: spec/game_spec.rb [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+      allow(game).to receive(:matrix)
+        .and_return([%w[_ x o], %w[o x o], %w[_ x _]])
+      expect(game.check_vertical).to eql(true)
+    end
+    it 'should return true if the slots match vertically from third column' do
```
Comment (bellom) [inline spec/game_spec.rb:57]: **Another one !**
Leave one empty line between feature, context or describe blocks. Do not leave empty line after the last such block in a group. [checkout this article to fix the issue](https://github.com/rubocop-hq/rspec-style-guide#empty-lines-between-describes)

**Review by bellom — CHANGES_REQUESTED** [review]
Hi Team, 
Great job using `Git Flow`, It was a nice test you wrote, i check if specs are failing if i break some code in tested method, yes it did fail, awesome !.
**Just two things left to do;**
1. include empty space around `it` block. 
2. fix linter (checkout this [link](https://github.com/microverseinc/linters-config#troubleshooting) for help.)
Kindly message me on slack if you still need help;
_Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you_

File: spec/game_spec.rb (hunk 1 above)
Comment (KaushikShivam) [reply spec/game_spec.rb:51]: We've fixed the suggested changes.

File: spec/game_spec.rb (hunk 2 above)
Comment (KaushikShivam) [reply spec/game_spec.rb:57]: Done

**Review by bellom — APPROVED** [review]
Hi Team, 
I can see your hard work on this project, don't forget to use `Git Flow` in the next project.
Go head and merge :+1:

**Pull request merged**